app.js

```diff
@@ -1,9 +1,18 @@
+const {getTopics, getEndpoints} = require("./controllers/topics.controllers")
 const express = require("express")
 const app = express()
 app.use(express.json())
 
-const {getTopics} = require("./controllers/topics.controllers")
+
 
 app.get('/api/topics', getTopics)
 
-module.exports = app
\ No newline at end of file
+app.get('/api', getEndpoints)
+
+module.exports = app
+
+// const port = 8080;
+
+// app.listen(port, () => {
+//   console.log(`Server is running on port ${port}`);
+// });
```

Comment:
Make sure to remove any commented out code before submitting work to us. When you submit a pull request it should be what you deem to be your final work and should be presented as such, so commented out or redundant lines like these should be removed in advance.

On a side note: If you wish to have a `.listen` method in place then you won't be able have it in this file, it'll need be in a different file.